OpenRA.Mods.Common/Projectiles/Bullet.cs

```diff
@@ -174,10 +178,23 @@ public Bullet(BulletInfo info, ProjectileArgs args)
 
 			if (info.ContrailLength > 0)
 			{
+				hasContrail = true;
+
+				// Approximate maximum contrail length in WDist, used to calculate ScreenMap bounds.
```

Comment:
Wouldn't it be better to update the `ContrailRenderable` to return a proper bounding rect, and then intersect that with the sprite bounds?

Reply:
Probably, but judging by the compile errors I get after 2 hours of working on this, it looks like I have no clue how to do that.